plugins/filter_aws/aws.h

```diff
@@ -112,9 +134,10 @@ struct flb_filter_aws {
     /* e.g.: if tag_is_enabled[0] = FALSE, then filter aws should not inject first tag */
     int *tag_is_enabled;
 
-    /* number of new keys added by this plugin */
-    int new_keys;
```

Comment:
was this just not used before? Or is there something new that replaces

Reply:
Yes, I noticed that it wasn't really used, therefore I removed this attribute. (Feel free to double check.)